docs/tutorials/geomesa-hbase-s3-on-aws.rst

```diff
@@ -107,7 +126,7 @@ To start executing SQL queries using Spark over your GeoMesa on HBase on S3 clus
 
 .. code-block:: shell
     
-    $ JARS=file:///usr/lib/hbase/hbase-common.jar,file:///usr/lib/hbase/hbase-client.jar,file:///usr/lib/hbase/hbase-protocol.jar,file:///usr/lib/hbase/hbase-server.jar,file:///opt/geomesa/dist/spark/geomesa-hbase-spark-runtime_2.11-${VERSION}.jar,file:///usr/lib/hbase/conf/hbase-site.xml
+    $ JARS=file:///opt/geomesa/dist/spark/geomesa-hbase-spark-runtime_2.11-${VERSION}.jar,file:///usr/lib/hbase/conf/hbase-site.xml
```

Comment:
this is what we do with accumulo - one jar with the hbase embedded. downside is it is not cluster specific. they get the version that comes with geomesa